src/PlusDataCollection/OptiTrack/vtkPlusOptiTrack.h

```diff
@@ -75,7 +75,7 @@ class vtkPlusDataCollectionExport vtkPlusOptiTrack : public vtkPlusDevice
 
 private: // Variables
   /*! Index of the last frame number. This is used for providing a frame number when the tracker doesn't return any transform */
-  unsigned long FrameNumber;
+  unsigned long FrameNumber = 0;
```

Comment:
I'm not sure if all of our supported compilers allow this. Anyway, probably we won't need this variable.

Reply:
Good point, removed!